app/models/comment.rb

```diff
@@ -0,0 +1,9 @@
+class Comment < ApplicationRecord
+  belongs_to :post, class_name: 'Post', foreign_key: :post_id
+  belongs_to :author, class_name: 'User', foreign_key: :author_id
+
+  def update_comment_counter(id)
+    new_counter = Comment.where(post_id: id).count
+    Post.where(id: id).update(comments_counter: new_counter)
+  end
```

Comment:
- [ ] kindly work on all the methods that are performing increments, passing a parameter to a method that will not be used elsewhere becomes unnecessary. We can also make use of the active record callback methods like `after_save` or `after_update`, and hide our `update_comment_counter` method under private, something like so:
```
after_save: update_comment_counter

private
def update_comment_counter
    author.increment!(:domment_counter)
 end
```